hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/PlaceholderTssLibrary.java

```diff
@@ -58,60 +58,60 @@ public PlaceholderTssLibrary() {
         this(DEFAULT_THRESHOLD);
     }
 
-    @NotNull
+    @NonNull
     @Override
-    public TssMessage generateTssMessage(@NotNull TssParticipantDirectory tssParticipantDirectory) {
+    public TssMessage generateTssMessage(@NonNull TssParticipantDirectory tssParticipantDirectory) {
         return null;
     }
 
-    @NotNull
+    @NonNull
```

Comment:
It states `@NonNull` here, but then the method returns `null`. This does not look correct to me.
There are more cases like this in this class (see warnings in IDE).

Reply:
This is wrong.

Reply:
Hey @jjohannes, As far as I understand, this was created as a temporary workaround.
All the uses of this class should be replaced with the library in `hedera-cryptography`.
cc: @MiroslavGatsanoga and @Neeharika-Sompalli

Reply:
This has been resolved in #16406